salt/cloud/clouds/vmware.py

```diff
@@ -2350,6 +2393,33 @@ def create(vm_):
             )
             clone_spec.customization = custom_spec
 
+        if customization and (devices and 'network' in list(devices.keys())) and 'Windows' in object_ref.config.guestFullName:
+            global_ip = vim.vm.customization.GlobalIPSettings()
+
+            if 'dns_servers' in list(vm_.keys()):
+                global_ip.dnsServerList = vm_['dns_servers']
+
+            identity = vim.vm.customization.Sysprep()
+            identity.guiUnattended = vim.vm.customization.GuiUnattended()
+            identity.guiUnattended.autoLogon = False
```

Comment:
Shouldn't this be set to ``True``? Why set this to ``False?`` Wouldn't you want the process to automatically log in as Administrator using the password ``win_password`` that has been specified?